meson.build

```diff
@@ -0,0 +1,82 @@
+project('exif', 'c', license: 'lgpl', version: '0.6.22')
```

Comment:
The license string should be from one from this list: https://spdx.org/licenses/